content/applications/inventory_and_mrp/manufacturing/workflows/subcontracting_dropship.rst

```diff
@@ -0,0 +1,263 @@
+ends with receiving the product into inventory.
+
+Create an SO
+------------
+
+It is only necessary to complete this step if the product is being purchased from the subcontractor
+to fulfill a customer need. If the product is being purchased to increase the quantity of stock
+on-hand, move on to the next step.
+
+To create a new |SO|, navigate to :menuselection:`Sales app --> Orders --> Orders`, and click
+:guilabel:`New`.
+
+Select the customer in the :guilabel:`Customer` drop-down menu. Then, click :guilabel:`Add a
+product` on the :guilabel:`Order Lines` tab, select the product in the :guilabel:`Product` drop-down
+menu, and enter a quantity in the :guilabel:`Quantity` field.
+
+Click :guilabel:`Confirm` to confirm the |SO|, at which point a :guilabel:`Purchase` smart button
+appears at the top of the page. This is the *subcontractor* |PO|, or the |PO| created to purchase
+the subcontracted product from the subcontractor.
+
+.. note::
+   An |SO| for the product only creates a *subcontractor* |PO| if the *Replenish on Order (MTO)*
+   route is enabled on the product's page, **and** there is no stock of the product on-hand.
+
+   If there is stock on-hand, confirming an |SO| for the product will instead create a delivery
+   order, because Odoo assumes that the |SO| is fulfilled using the stock in the warehouse.
+
+   This is not the case for subcontracted products that are dropshipped to the end customer. In that
+   case, a *subcontractor* |PO| is **always** created, even if there is stock on-hand.
+
+Process subcontractor PO
+------------------------
+
+If a *subcontractor* |PO| was not created in the previous step, do so now by navigating to
+:menuselection:`Purchase app --> Orders --> Purchase Orders`, and clicking :guilabel:`New`.
+
+Begin filling out the |PO| by selecting a subcontractor from the :guilabel:`Vendor` drop-down menu.
+
+In the :guilabel:`Products` tab, click :guilabel:`Add a product` to create a new product line.
+Select a product produced by the subcontractor in the :guilabel:`Product` field, and enter the
+quantity in the :guilabel:`Quantity` field.
+
+Finally, click :guilabel:`Confirm Order` to confirm the *subcontractor* |PO|.
+
+When a |PO| is confirmed for a product that requires dropshipping components to a subcontractor, a
+receipt or dropship order is automatically created, and can be accessed from the corresponding
+:guilabel:`Receipt` or :guilabel:`Dropship` smart button that appears at the top of the |PO|.
+
+.. image:: subcontracting_dropship/subcontractor-po.png
+   :align: center
+   :alt: A subcontractor PO for a *Dropship Subcontractor on Order* product, with a Receipt smart
+         button at the top of the page.
+
+In addition, an |RfQ| is created for the components that are purchased from the vendor and sent to
+the subcontractor. However, the |RfQ| **IS NOT** automatically linked to the *subcontractor* |PO|.
```

Comment:
I *think* the standard for all caps in the negative is just to do the negative word (not both the positive + negative). Open to interpretation ofc 😅 

```suggestion
the subcontractor. However, the |RfQ| is **NOT** automatically linked to the *subcontractor* |PO|.
```